### ISSUE TITLE
telegraf_install_url is being ignored

### ISSUE THREAD

**Issue by rbrutas** [rossmcdonald/telegraf#10]
When i set the telegraf_install_url value on the role, ansible ignores it.

role:

```
- role: rossmcdonald.telegraf
      telegraf_install_url: http://some_url.com/telegraf_1.0.0-beta3_amd64.deb
      telegraf_hostname: ""
```

ansible output:

```
TASK [rossmcdonald.telegraf : Download Telegraf package via URL [Debian/Ubuntu]] ***
       task path: /tmp/kitchen/roles/rossmcdonald.telegraf/tasks/install-debian.yml:34
       skipping: [localhost] => {"changed": false, "skip_reason": "Conditional check failed", "skipped": true}

       TASK [rossmcdonald.telegraf : Install downloaded Telegraf package [Debian/Ubuntu]] ***
       task path: /tmp/kitchen/roles/rossmcdonald.telegraf/tasks/install-debian.yml:38
       skipping: [localhost] => {"changed": false, "skip_reason": "Conditional check failed", "skipped": true}
```

**Comment by rbrutas** [comment]
i added this to tasks/install-debian.yml

```
- name: Echo the content of telegraf_install_url
  debug:
    msg: "{{ telegraf_install_url }}"
```

and not getting any value

```
       TASK [rossmcdonald.telegraf : Echo the content of telegraf_install_url] ********
       task path: /tmp/kitchen/roles/rossmcdonald.telegraf/tasks/install-debian.yml:14
       ok: [localhost] => {
           "msg": null
       }
```

**Comment by rbrutas** [comment]
fixed by cd50e3f
